index.js

```diff
@@ -240,6 +244,60 @@ class ServerlessCustomDomain {
         throw Error(`Could not find the certificate ${certificateName}`);
       }
       return certificateArn;
+    }).catch((err) => {
+      // No cert found in the ISSUED status. Let's look through certs in all other statuses.
+      const invalidCertArn = acm.listCertificates({ CertificateStatuses: [
```

Comment:
The reason was so that you could easily see the INVALID list in the console.log vs the VALID list above in the previous console.log.

Originally I was thinking this would be useful for people with a lot of certs, however, after a little more thought, it also makes sense to just make an additional call to get the status about the cert in question rather than making the user figure it out by comparing the lists or having to login to AWS.